terraform-aws-github-runner/modules/runners/lambdas/runners/src/lambda.test.ts

```diff
@@ -55,6 +65,47 @@ describe('scaleUp', () => {
     expect(callback).toBeCalledTimes(1);
     expect(callback).toBeCalledWith('Failed handling SQS event');
   });
+
```

Comment:
Maybe it's worth adding the test for the SQS event that mixes recoverable and non-recoverable errors? ([related](https://github.com/pytorch/test-infra/pull/1947/files#r1091142852))